MakariiHlyva/HW4/main.py

```diff
@@ -0,0 +1,19 @@
+#-------------Calculate factorial of natural number, which will be entered by user, without using recurcion----------------
+
```

Comment:
try to implement this task without import math

Reply:
may I use loop in this task?